scripting/whats-new/What-s-New-in-Windows-PowerShell-50.md

```diff
@@ -15,7 +15,10 @@ Windows PowerShell ® 5.0 includes significant new features that extend its use,
 
 Windows PowerShell 5.0 is backward-compatible. Cmdlets, providers, modules, snap-ins, scripts, functions, and profiles that were designed for Windows PowerShell 4.0, Windows PowerShell 3.0, and Windows PowerShell 2.0 generally work in Windows PowerShell 5.0 without changes.
 
-Windows PowerShell 5.0 is installed by default on Windows Server ® 2016 Technical Preview and Windows 10 ®. To install Windows PowerShell 5.0 on Windows Server 2012 R2, Windows 8.1 Enterprise, or Windows 8.1 Pro, download and install [Windows Management Framework 5.0](http://aka.ms/wmf5download). Be sure to read the download details, and meet all system requirements, before you install Windows Management Framework 5.0.
+# Installing Windows PowerShell
```

Comment:
:+1: great suggestions. i started digging into it as well, i think there's a lot of improvements to be made here... so many that i am going to tap out and open a new issue: #712